alyx/alyx/urls.py

```diff
@@ -34,6 +34,8 @@
 urlpatterns = [
     path('', mv.api_root),
 
+    path('', include('experiments.urls')),
```

Comment:
is the '' normal here?

Reply:
yup I decided to keep the urls in the app itself for easier refactoring. 
Do you think it will cause issues ?

Reply:
I meant the first argument to `path()` -- but actually I see that the URLs are properly named in `alyx/experiments/urls.py` so we're good!